src/internal/kopia/upload.go

```diff
@@ -960,14 +960,16 @@ func inflateBaseTree(
 		return nil
 	}
 
+	ctx = clues.Add(ctx, "snapshot_root_id", snap.ID)
```

Comment:
This is the ID of the base snapshot, not the ID of the root item in the base snapshot
```suggestion
	ctx = clues.Add(ctx, "snapshot_base_id", snap.ID)
```